.drone.yml

```diff
@@ -15,16 +19,19 @@ pipeline:
     when:
       event: [push, pull_request]
 
-  # @todo #345:60m Use ready image for drone npm step. stb2
-  #  Image `fidals/se-nodejs:dev` already contains built static and node_modules.
   npm:
-    image: node:slim
+    image: docker/compose:1.17.1
     environment:
       - DEPS_DIR=/drone/deps
     commands:
-      - npm install
-      - npm install -g gulp-cli
-      - gulp build
+      - cd docker
+      - cp drone_env/* env_files/
+      # Build python images with sources and static files
+      - docker-compose -f docker-compose-build.yml build --no-cache nodejs
```

Comment:
Let's  build nodejs in the `docker-build:` and move the pipe before the `test`

Reply:
@artemiy312 , it's for `gulp build` step below

Reply:
@artemiy312 , python-prod's build uses result of `dc run --rm nodejs gulp build`.
So, seems we are needed in separated npm step with `npm install` and `gup build` inside